compiler/util-klib-abi/tests-gen/org/jetbrains/kotlin/library/abi/KlibDumpParserTestGenerated.java

```diff
@@ -0,0 +1,187 @@
+/*
```

Comment:
Now (9f259b975cedfc268784092a392340664d3b9ca6) these tests are automatically generated during the build and they no longer should be committed to the repo.